### ISSUE TITLE
Unsure about the role of the `-r` option

### ISSUE THREAD

**Issue by ngirard** [GarkGarcia/icon-pie#12]
According to your docs, there are 3 possible re-sampling filters, *nearest*, *linear* and *cubic*.

- With SVGs, the filter is always *linear*, so the `-r` option doesn't apply.

- With raster graphics, you seems to provide a sane default (*nearest*), but it doesn't save the user from thinking about which one is the best for his needs, since you're giving him the choice. Unfortunately, no further help is given to help him to decide. Why would I choose the *cubic* filter ? Based on what criteria ?

So it seems to me, like there's room for rethinking why and how you're giving users the choice between re-sampling filters.

**Comment by GarkGarcia** [comment]
Makes sense. Support for linear interpolation is an important feature, but cubic interpolation doesn't have any big use case and nearest-neighbor is already the default.

**Comment by ngirard** [comment]
Forgive me for the misleading title of this report. I didn't mean to say that the option should necessary disappear ; but that, *if* the user is given the choice, he should also be guided in his decision.

Earlier today I found [this](https://superuser.com/questions/1093805/how-to-achieve-high-quality-downscaling-of-diagrams-like-in-firefox) discussion about different filters for downsampling, which you may find of interest.

What I don't know, is if there's an authoritative source of information which would answer the following question: 

*When downsampling images to produce application icons, is there a filter that is always the best ? If not, what filter to choose, based on which criteria ?*
